crates/ruff_linter/src/rules/pyupgrade/rules/unnecessary_encode_utf8.rs

```diff
@@ -256,3 +263,78 @@ pub(crate) fn unnecessary_encode_utf8(checker: &Checker, call: &ast::ExprCall) {
         _ => {}
     }
 }
+
+fn string_contains_string_only_escapes(string: &StringLiteralValue, locator: &Locator) -> bool {
+    for fragment in string {
+        let flags = fragment.flags;
+
+        if flags.prefix().is_raw() {
+            continue;
+        }
+
+        let total_len = fragment.end() - fragment.start();
+        let value_len = total_len - flags.opener_len() - flags.closer_len();
+
+        if value_len > fragment.as_str().text_len() {
+            return string_fragment_contains_string_only_escapes(fragment, locator);
+        }
```

Comment:
I don't think this is correct. It will always return `false` if `string_fragtment_contains_string_only_escapes` returns `false` for the first string where `value_len > fragtment.as_str().text_len()` is true

```py
a = "test \
more" "a\\ escaped"
```